flink-libraries/flink-table/src/test/scala/org/apache/flink/table/api/stream/table/OverWindowTest.scala

```diff
@@ -203,19 +203,17 @@ class OverWindowTest extends TableTestBase {
         "DataStreamCalc",
         unaryNode(
           "DataStreamOverAggregate",
-          unaryNode(
-            "DataStreamCalc",
-            streamTableNode(0),
-            term("select", "a", "c", "proctime")
-          ),
+          streamTableNode(0),
```

Comment:
The new plan is less efficient, because a projection isn't pushed down anymore and a wider row needs to be materialized in the over window.